rhino/src/main/java/org/mozilla/javascript/AbstractEcmaObjectOperations.java

```diff
@@ -249,6 +249,25 @@ static void put(Context cx, Scriptable o, int p, Object v, boolean isThrow) {
         }
     }
 
+    /**
+     * Set ( O, P, V, Throw)
+     *
+     * <p><a href="https://262.ecma-international.org/12.0/#sec-set-o-p-v-throw">7.3.4 Set (O, P, V,
```

Comment:
The spec says we have to return a boolean - i know that we are not able to do the right thing in many cases - and this was a major problem during the implementation of the proxy stuff.
But maybe we can try to follow the spec here at least for new methods (even if we can't return always the right value)

What do you think?